### DISCUSSION TITLE
iOS App with iCloud or Onedrive sync

### DISCUSSION THREAD

**Discussion by amit510** [nuttyartist/notes#362] [Ideas]
Just wanted to see if this is something planned for the future, to have a iOS app with sync functionality. This would be an awesome addition! 

Thank you!

**Comment by nuttyartist** [comment]
Hi! Indeed it would be great, we just need more resources/coders to achieve this. I'd be more than happy If someone volunteers.

**Comment by pa-0** [comment]
Just throwing this out there, but a GitHub integration (where a user would be able to create new repos from the app, and sync their notes directly with the repo) would be pretty great as a cross-platform solution (for users who do not use onedrive or icloud)

**Comment by nuttyartist** [comment]
Hi there, this is quite against are objective of being friendly to non-technical end users. But there's an issue about supporting an arbitrary folder with plain text files rather than using a database, so that will allow easy syncing via GitHub etc.
